server/service/src/sync/translations/invoice_line.rs

```diff
@@ -87,6 +87,9 @@ pub struct LegacyTransLineRow {
     #[serde(deserialize_with = "empty_str_as_option_string")]
     #[serde(rename = "om_item_variant_id")]
     pub item_variant_id: Option<String>,
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    #[serde(rename = "linked_trans_line_ID")]
+    pub linked_invoice_id: Option<String>,
```

Comment:
Should it not be linked_invoice_line_id everywhere?

Reply:
Oh this should be linked transact id! XD